recipes/aospy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "aospy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/27/39/d6674839265bbdccbd97291857baddf636e4329e03b6c179bcc62d390970/aospy-0.1.tar.gz
+  md5: 4201285d7171be95fa016039cceca719
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7
```

Comment:
Does this package relies on the numpy ABI? If so we may need to use `numpy x.x` at both `build` and `run`.